adagucserverEC/CImageDataWriter.cpp

```diff
@@ -3356,10 +3357,7 @@ int CImageDataWriter::end(){
   
   //Output WMS getmap results
   if(errorsOccured()){
-      
-    
-    CDBError("Errors occured during image data writing");
```

Comment:
Are errors printed CREPORT_ERROR_NODOC shown in ADAGUC error reporting if report functionality is not enabled? E.g. is older behavior unchanged?

Reply:
Yes. Wrote a test to verify. Commit a484eb4677fe318b57decea33b506f547f31cebe